build.gradle

```diff
@@ -21,12 +21,14 @@ buildscript {
     }
     dependencies {
         classpath('io.swagger.codegen.v3:swagger-codegen:3.0.52')
+        // Required for gradle liquibase plugin
+        classpath ('org.liquibase:liquibase-core:4.26.0')
```

Comment:
Note for reviewers:
- Adding this classpath to the buildscript is new requirement with gradle liquibase 3.0.0
- Ideally, we wouldn't pin the version of liquibase-core. However, the version appears to be required when defining the classpath. Additionally, we need to pin the version for the implementation of liquibase-core, or it would default to 4.24.0.

Reply:
Good shout -- I will revisit this as part of our latest bulk version update to see if it's still needed.

When we upgrade Spring Boot to latest -- 3.3.2 -- we'll get liquibase versioned at 4.27.0, do you think that might allow us to remove the version here? https://github.com/spring-projects/spring-boot/blob/v3.3.2/spring-boot-project/spring-boot-dependencies/build.gradle#L1066

Reply:
I'm not sure, it wouldn't build without a version here. But, maybe I was just missing something.